meinberlin/templates/base.html

```diff
@@ -45,12 +45,14 @@
     {% if tracking %} {% tracking_code %} {% endif %}
 </head>
 <body>
-    <a href="#main" class="visually-hidden-focusable">{% translate "Skip to content "%}</a>
+    <div class="visually-hidden-focusable">
```

Comment:
no this was actually just advice from link from lighthouse when testing the list, the example had the link wrapped with a div styled to not be displayed instead of applying styles direct to the link, also the addition of tabindex to the destination element

Reply:
Although we do have to change the `visually-hidden-focusable`as it's from bootstrap